src/main/kotlin/org/pkl/intellij/annotator/PklModuleUriAndVersionAnnotator.kt

```diff
@@ -255,27 +225,56 @@ class PklModuleUriAndVersionAnnotator : PklAnnotator() {
     element: PklModuleUri,
     reference: PsiReference,
     resolved: List<PsiElement>,
-    holder: AnnotationHolder
-  ) {
-    if (!element.stringConstant.content.text.startsWith("@")) return
-    if (element.isInPklProject || element.isInPackage) return
-    if (!element.isPhysical) return
-    if (element.containingFile.virtualFile.fileSystem !is LocalFileSystem) {
-      return
+    holder: AnnotationHolder,
+    context: PklProject?
+  ): Boolean {
+    if (!element.stringConstant.content.text.startsWith("@")) return false
+    if (!element.isPhysical) return false
+    if (resolved.isNotEmpty() && resolved.all { it.isInPackage }) return false
```

Comment:
Nit (I half expect all the branching to be JITed out, but still)

```suggestion
    if (!element.stringConstant.content.text.startsWith("@") ||
        !element.isPhysical ||
        resolved.isNotEmpty() && resolved.all { it.isInPackage })
      return false
```

Reply:
I think this is a code style difference? I find the current form easier to read than your suggestion.